### PR TITLE
fix: Use mailbox templates, solve formatting problems, and optimize some interfaces

### PR DESCRIPTION
**Pull request by ZhaoKang0610**
### Description

Fixes # (issue numbers)

### Checklist

- [x] I have explained the need for this PR and the problem it solves
- [ ] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [ ] I have updated the documentation to reflect this change
- [ ] I have verified that this change is backward compatible

### CODE REVIEW

File: apps/workbench/backend/api/account/account.go [hunk 1]
```diff
@@ -2,6 +2,7 @@ package account
 
 import (
 	"fmt"
```
Comment (yuluo-yx) [inline apps/workbench/backend/api/account/account.go:4]: adjust the order pkg @ZhaoKang0610 

pls check other file.

File: apps/workbench/backend/api/developer/developer.go [hunk 2]
```diff
@@ -111,7 +116,7 @@ func VerifyEmailAndPwd(ctx context.ApplicationContext) gin.HandlerFunc {
 		validate := validator.New()
 		err = validate.RegisterValidation("verifyPwd", verifyPwd)
 		if err != nil {
-			c.AbortWithStatusJSON(http.StatusInternalServerError, types.NewValidResponse(&Resp{
```
Comment (yuluo-yx) [inline apps/workbench/backend/api/developer/developer.go:114]: IMO, the `http.StatusBadRequest` status code used here cannot well represent the status of the service. Custom business exception codes should be used.

ple see https://www.rfc-editor.org/rfc/rfc7231.html#section-6.5.1

File: apps/workbench/backend/api/developer/developer.go [hunk 3]
```diff
@@ -358,16 +415,15 @@ func VerifyPwdVerificationCode(ctx context.ApplicationContext) gin.HandlerFunc {
 		}
 		// Verify that the password check and email check have passed
 		step, _ := db.RedisDB.HGet(developer.GetEmailHashStr(req.Email), "mod-pwd-step").Int()
-		if step != 2 {
+		if step != 1 {
 			c.AbortWithStatus(http.StatusBadRequest)
 			return
 		}
-		dlp := &developer.Developer{
```
Comment (yuluo-yx) [inline apps/workbench/backend/api/developer/developer.go:365]: this change is not needed?

File: apps/workbench/backend/auth/jwt/jwt.go [hunk 4]
```diff
@@ -53,21 +54,28 @@ func ParseToken(tokenString string) (*MyClaims, error) {
...
-
-	claims, ok := token.Claims.(*MyClaims)
-	if !ok || !token.Valid {
-		return "", fmt.Errorf("invalid token")
+	if claims, ok := token.Claims.(*MyClaims); ok && token.Valid {
+		return time.Now().Unix() < claims.ExpiresAt, nil
 	}
 
-	return claims.ID, nil
+	return false, fmt.Errorf("Invalid token")
```
Comment (yuluo-yx) [inline apps/workbench/backend/auth/jwt/jwt.go:69]: Is use error.New("info")?

File: apps/workbench/backend/router/router.go [hunk 5]
```diff
@@ -21,28 +21,37 @@ func NewRouter(ctx context.ApplicationContext) http.Handler {
 }
 func mountAPIs(e *gin.Engine, ctx context.ApplicationContext) {
 	api := e.Group("v1")
-	// api.Any("/*", middleware.JWTAuthMiddleware())
 	{
     api.POST("/signup", account.SignUp(ctx))
 		api.POST("/signup_verify", account.SignUpVerify(ctx))
 		api.POST("/signin", account.SignIn(ctx))
-		api.GET("/logout", middleware.JWTAuthMiddleware(), account.Logout())
```
Comment (yuluo-yx) [inline apps/workbench/backend/router/router.go:29]: We should reduce unnecessary changes in a public relations and maintain the specificity of public relations.

File: apps/workbench/backend/service/mail/mail.go [hunk 6]
```diff
@@ -1,19 +1,25 @@
...
+	"os"
+	"path"
+	"runtime"
 	"time"
+
+	"github.com/Boostport/mjml-go"
+	"github.com/jordan-wright/email"
 )
 
 var (
```
Comment (yuluo-yx) [inline apps/workbench/backend/service/mail/mail.go:20]: Extracted into public constants.

File: apps/workbench/backend/service/mail/mail_test.go [hunk 7]
```diff
@@ -3,13 +3,35 @@ package mail
...
 	"fmt"
+	"github.com/stretchr/testify/assert"
 	"testing"
 )
 
 func TestMailService_SendMail(t *testing.T) {
 	ctx := context.Background()
 	ms := mailService{}
-	email := "zhaokang@deamgo.com"
+	email := "zhaokang0610@gmail.com"
```
Comment (yuluo-yx) [inline apps/workbench/backend/service/mail/mail_test.go:13]: Extracted into config file.

File: apps/workbench/backend/service/mail/mail.go (hunk 6 above)
Comment (ZhaoKang0610) [reply apps/workbench/backend/service/mail/mail.go]: ok

File: apps/workbench/backend/router/router.go (hunk 5 above)
Comment (ZhaoKang0610) [reply apps/workbench/backend/router/router.go:29]: This change makes the code more standardized

File: apps/workbench/backend/auth/jwt/jwt.go (hunk 4 above)
Comment (ZhaoKang0610) [reply apps/workbench/backend/auth/jwt/jwt.go]: OK

File: apps/workbench/backend/api/developer/developer.go (hunk 3 above)
Comment (ZhaoKang0610) [reply apps/workbench/backend/api/developer/developer.go]: Make sense

File: apps/workbench/backend/api/account/account.go (hunk 1 above)
Comment (ZhaoKang0610) [reply apps/workbench/backend/api/account/account.go]: ok

File: apps/workbench/backend/api/developer/developer.go (hunk 2 above)
Comment (ZhaoKang0610) [reply apps/workbench/backend/api/developer/developer.go]: This is indeed the server's error, which may occur for the structural binding password verification.

File: apps/workbench/backend/service/mail/mail_test.go (hunk 7 above)
Comment (ZhaoKang0610) [reply apps/workbench/backend/service/mail/mail_test.go:13]: unnecessary

**Review by yuluo-yx — APPROVED** [review]
LGTM
